account_invoice_section_sale_order/views/res_config_settings.xml

```diff
@@ -28,6 +28,14 @@
                         />
                         <field name="invoice_section_name_scheme" />
                     </div>
+                    <div class="row">
+                        <label
+                            for="always_create_invoice_section"
+                            class="col-lg-4 o_light_label"
+                            string="When to create sections"
```

Comment:
Can you clarify in the UI that the option concerns invoices for a single sale order?